UnityProject/Assets/Scripts/Items/Weapons/Melee/StunBatonV2.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using AddressableReferences;
+using Mirror;
+using UnityEngine;
+
+[RequireComponent(typeof(Pickupable))]
+public class StunBatonV2 : NetworkBehaviour, ICheckedInteractable<HandActivate>, IServerSpawn
+{
+	public SpriteHandler spriteHandler;
+
+    public MeleeStun meleeStun;
```

Comment:
same here. Make it private and get the components in an Awake method

Reply:
Fixed in fe4b08d. Set to private.